app/_how-tos/synchronize-with-git.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Synchronize Insomnia content with Git
```

Comment:
Since this works with many version control providers do you think it would make sense to use the word version control instead of git? to cover the bitbucket stuff?

Reply:
Bitbucket and Azure DevOps are also Git providers, so I think only Git is supported in Insomnia

Reply:
I was reading content for storage options and learned that git sync is a Team tier and above feature (https://insomnia.rest/pricing). Could you add a badge for the Team tier of Insomnia?